clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -215,8 +215,10 @@ check_sane_control_flow()
             gen_marker(1, TRACE_MARKER_TYPE_FILETYPE, OFFLINE_FILE_TYPE_ENCODINGS),
 #    if defined(X86_64) || defined(X86_32)
             // 0x74 is "je" with the 2nd byte the offset.
+            gen_instr_encoded(0x71019dba, { 0x20 }),
```

Comment:
Why do we need these changes in this test?

Reply:
Removed them.